www/src/routes/Landing.tsx

```diff
@@ -276,7 +276,21 @@ const Footer: FC = () => {
     return (
         <Grid container item className={classes.footer}>
             <Grid container item className={classes.wave_pattern} justify='center' direction='column' alignItems='center'>
-                <img src={LightGreenLogo} className={classes.footer_logo} />
+                <Grid container item alignItems='center' justify='center' xs={11}>
+                    <Grid item xs={4}>
+                        <Typography className={classes.footer_text} align='left'>
+                            Contact: external@compeclub.com
+                        </Typography>
+                    </Grid>
+                    <Grid container item xs={4} alignItems='center' justify='center'>
+                        <img src={LightGreenLogo} className={classes.footer_logo} />
+                    </Grid>
+                    <Grid item xs={4}>
+                        <Typography className={classes.footer_text} align='right'>
+                            Curious how we&apos;re building CompE+? Checkout the <a href='https://github.com/UAlbertaCompEClub/compe-plus'>Github</a> 😎
```

Comment:
It might be just me but I think I'd prefer _Checkout the Github repo_ or _Checkout the repo in Github_ inplace of Checkout the Github 😆

Reply:
+1